maven-core/src/test/java/org/apache/maven/graph/DefaultGraphBuilderTest.java

```diff
@@ -390,6 +400,7 @@ private Dependency toDependency( MavenProject mavenProject )
         private String resumeFrom = "";
         private String makeBehavior = "";
         private File requestedPom = new File( PARENT_MODULE, "pom.xml" );
+        private boolean isRecursive = true;
```

Comment:
Shouldn't this just be `recursive`?

Reply:
Yep, I agree. It's changed now, and I've found and fixed some other inconsistencies around that parameter as well. Ready to be reviewed again.